package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@next/next-common-files",
+  "version": "0.0.0",
+  "description": "Loaders for common file types.",
+  "main": "./index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest"
```

Comment:
We are missing some tooling, such as `standard-release` and `commitlint`. Please check https://github.com/moxystudio/next-runtime-env/blob/master/package.json and add what we are missing.